corehq/pillows/mappings/user_mapping.py

```diff
@@ -1,4 +1,4 @@
-USER_INDEX="hqusers_31n2pdyzqix4hd023lv6h67771c7wer"
+USER_INDEX="hqusers_31n2pdyzqix4hd023lv6h67771c7wes"
```

Comment:
can you generate a new guid? visually it's really hard to tell these are different